vault.md

```diff
@@ -43,7 +43,55 @@ kubectl -n vault exec vault-0 --  vault kv get kv/hello
 kubectl -n vault exec vault-0 --  vault kv delete kv/hello
 ```
 
-- To uninstall it
+
+### Using vault custom script and vault cli
+
+There is a script in the script folder ready to install and configure vault.
+
+1 - Install vault in the kind cluster.
+
+Go to the scripts folder and launch the script from its location:
+````bash
+./vault.sh
+````
+
+Copy the Root Token shown in the output.
+
+2- Authenticate as Root.
+````bash
+vault login
+````
+Note: paste the Root Token when prompted.
+
+
+3- Create a policy giving access to the path that will stock the secrets
+```bash
+cat <<EOF | vault policy write vault-quickstart-policy -                                                                                                                  ✔  13:07:05
+path "kv/*" {
+  capabilities = ["read", "create"]
+}
+EOF
+```
+
+4- Enable the userpass auth secret engine, and create user bob with access to the vault-quickstart-policy
+````bash
+vault auth enable userpass
+vault write auth/userpass/users/bob password=sinclair policies=vault-quickstart-policy
```

Comment:
I propose to script such commands using new functions: `enableUserPasswordAuth`, `registerUser` and `setUserPolicy` where we could pass a user/pwd and key path as parameters. WDYT ?